.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
on reset container getting the same error "workspace does not exist, rerun as root"

Reply:
Weird, never faced this.